README.md

```diff
@@ -85,99 +118,128 @@ cd elchemy
 In order to increase readability it's advised not to use `ffi` calls if not necessary and always document and doctest them.
 
+
 ## But what about out of function macros? Like tests and `use Module`?
+
 Unfortunately you can't write any macros with `do..end` blocks yet. You can write any out of function code using an elixir inline code with:
+
+
 ```elm
 {- ex
-  code_here
+  *code_here*
 -}
 ```
-But it's a last resort solution and shouldn't ever be abused.
+
+But it is a last resort solution and shouldn't ever be abused.
+
 
 ## Can I define an Elixir macro in Elchemy?
+
 So you want to write an Elm-like code, that will manipulate Elixir code, which generates an Elixir code that manipulates Elixir code? How about no?
 
-## Do I need to have Elm installed to compile my `.elm` files with Elchemy?
-Elchemy uses Elm to typecheck your program. It is possible to use it without Elm on your machine, while it's not advised.
 
+## Do I need to have Elm installed to compile my `.elm` files with Elchemy?
 
-# Maturity of the project
+Elchemy uses Elm to typecheck your program. It is possible to use it without Elm on your machine, while it's not advised.
 
-- Parser - **99%** of Elm's syntax (see [elm-ast](https://github.com/Bogdanp/elm-ast/issues))
-- Compiler - **90%** (Sophisticated incremental compilation. No support for Windows yet though ([#287](https://github.com/wende/elchemy/issues/287)) also big reliance on unix tools ([#288](https://github.com/wende/elchemy/issues/288))
-- Elchemy-core - **95%** ( Everything covered except side effects and JSON Decoders) 
-- Interop with Elixir - **90%** - Purity tests ([#162](https://github.com/wende/elchemy/issues/162)) and handling of macro-heavy libraries ([#276](https://github.com/wende/elchemy/issues/276)) to go 
-- Ideology - **70%** - We've got a pretty solid idea of where Elchemy is going 
-- Documentation - **80%** - There are two tutorials and a complete Gitbook documentation. Few entrance level tutorials though
-- Elchemy-effects - **20%** - You can't and shouldn't write anything with side-effects in Elchemy yet. We're working on finding the best solution for effects that would fit both Elm's and Elixir's community (see [#297](https://github.com/wende/elchemy/issues/297) for more info)
-- Elchemy-core for Erlang VM - **5%** - Everything for os related tasks like filesystem, OTP goodies etc are yet to be done
-- Elchemy type checker - **15%** - Self-hosted elchemy type inference algorithm
 
 # Contributor credits:
 
 - Tomasz Cichociński - [@baransu](https://github.com/baransu)
 - Colin Bankier - [@colinbankier](https://github.com/colinbankier)
 - Nathaniel Knight - [@neganp](https://github.com/neganp)
+- [Elm-in-Elm](https://github.com/elm-in-elm/compiler) creator Martin Janiczek - [@janiczek](https://github.com/Janiczek)
```

Comment:
We don't use elm-in-elm at all. But it definitely is a good place to mention @Bogdanp for creating elm-ast which was the first dependency and the most influential inspiration in creating Elchemy

Reply:
Also, feel free to mention yourself @ShalokShalom as recently you've been giving Elchemy more love than I was able to deliver 😄 

OT: I've been rewriting type inference from scratch and there's been some great developments in term of having it clear and concise. Really stoked to get this to MVP stage 🎉